### PR TITLE
Correct typos in docstrings of the controlled rotation operations

### PR DESCRIPTION
**Pull request by co9olguy**
The controlled rotation gates have a number of typos in them. 

I've fixed the formula for CRY, but CRX and CRZ may still be incorrect (didn't have time to check)

### CODE REVIEW

File: pennylane/ops/qubit.py [hunk 1]
```diff
@@ -950,7 +950,7 @@ class CRX(Operation):
     .. math::
 
         \begin{align}
-            CRX(\phi) &= I_{1}\otimes RZ_{2}(\pi / 2) ~\cdot~ I_{1}\otimes RY_{2}(\phi/2) ~\cdot~ CNOT_{12} ~\cdot~ RY_{2}(-\phi/2) ~\cdot~ CNOT_{12} ~\cdot~ I_{1}\otimes RZ_{2}(-\pi / 2)\notag \\[10pt]
+            CR_x(\phi) &= I_{1}\otimes R{z2}(\pi / 2) ~\cdot~ I_{1}\otimes R{y2}(\phi/2) ~\cdot~ CNOT_{12} ~\cdot~ R{y2}(-\phi/2) ~\cdot~ CNOT_{12} ~\cdot~ I_{1}\otimes R{z2}(-\pi / 2)\notag \\[10pt]
```
Comment (josh146) [inline pennylane/ops/qubit.py:953]:
```suggestion
            CR_x(\phi) &= I_{1}\otimes R_{z2}(\pi / 2) ~\cdot~ I_{1}\otimes R_{y2}(\phi/2) ~\cdot~ CNOT_{12} ~\cdot~ R_{y2}(-\phi/2) ~\cdot~ CNOT_{12} ~\cdot~ I_{1}\otimes R_{z2}(-\pi / 2)\notag \\[10pt]
```

**Review by josh146 — APPROVED** [review]
Quick question: while this matches the convention we use for our single qubit gate formulas, I worry that `R_{y2}` might be a bit confusing (it could be read as a 'y2' rotation).

Is there a better way of indicating 'RY rotation acting on wire 2'? Something like `R_y^{(2)}`, or `R_{y_2}`?

**Comment by co9olguy** [comment]
> Quick question: while this matches the convention we use for our single qubit gate formulas, I worry that `R_{y2}` might be a bit confusing (it could be read as a 'y2' rotation).
> 
> Is there a better way of indicating 'RY rotation acting on wire 2'? Something like `R_y^{(2)}`, or `R_{y_2}`?

Good question. Leaving this issue open because I still think there are errors in the expressions (we'll need to verify). We can resolve the formatting question before merging

**Comment by josh146** [comment]
> Good question. Leaving this issue open because I still think there are errors in the expressions (we'll need to verify). We can resolve the formatting question before merging

@co9olguy just revisiting this PR before it becomes too stale. The `pi` -> `phi` typo is pretty major, so it would be nice to merge it in. 

> Is there a better way of indicating 'RY rotation acting on wire 2'? Something like R_y^{(2)}, or R_{y_2}?

Do you want to make an executive decision here? I'm happy with either of these options, or any other alternatives.

**Comment by co9olguy** [comment]
@josh146 since decompositions are now:
- prevalent in the code
- determined by device as first option
I believe it's better to remove these from the docstrings altogether (they only appear in text for these three specific operators from what I could tell). If we would like to display the decompositions, I'd rather have the code be the source of truth and populate the docs programmatically rather than manually (as you can see, prone to errors)

**Comment by josh146** [comment]
> I believe it's better to remove these from the docstrings altogether (they only appear in text for these three specific operators from what I could tell). If we would like to display the decompositions, I'd rather have the code be the source of truth and populate the docs programmatically rather than manually (as you can see, prone to errors)

:+1:

**Pull request merged**